pkg/asset/tls/certkey.go

```diff
@@ -104,6 +107,61 @@ func (c *CertKey) loadCertKey(f asset.FileFetcher, filenameBase string) (bool, e
 	return true, nil
 }
 
+// AssetExpirerInterface determines if a Certificate has expired.
+type AssetExpirerInterface interface {
```

Comment:
Nit: Can we call this something like ExpiredCertInterface or ExpiredCertChecker?